test_scripts/Defects/7_0/1917_Media_navigation_and_nonmedia_apps_must_be_activated_during_active_embedded_audio_source.lua

```diff
@@ -0,0 +1,159 @@
+common.getConfigAppParams(naviAppId).isMediaApplication = false
+common.getConfigAppParams(nonMediaAppId).appHMIType = { "DEFAULT" }
+common.getConfigAppParams(nonMediaAppId).isMediaApplication = false
+
+--[[ Local Functions ]]
+local function getHMIValues()
+  local params = hmi_values.getDefaultHMITable()
+  params.BasicCommunication.MixingAudioSupported.attenuatedSupported = true
+  return params
+end
+
+local function activateApp(pAppId)
+  if not pAppId then pAppId = 1 end
+  local requestId = common.getHMIConnection():SendRequest("SDL.ActivateApp", { appID = common.getHMIAppId(pAppId) })
+  common.getHMIConnection():ExpectResponse(requestId)
+  common.getMobileSession(pAppId):ExpectNotification("OnHMIStatus")
+  :ValidIf(function(_, data)
+    if config["application"..pAppId].registerAppInterfaceParams.isMediaApplication == false and
+      config["application"..pAppId].registerAppInterfaceParams.appHMIType[1] == "DEFAULT"
+      then
+        return data.payload.audioStreamingState == "NOT_AUDIBLE" and
+          data.payload.hmiLevel == "FULL" and
+          data.payload.systemContext == "MAIN"
+      else
+        return data.payload.audioStreamingState == "AUDIBLE" and
+          data.payload.hmiLevel == "FULL" and
+          data.payload.systemContext == "MAIN"
+    end
+  end)
+end
+
+local function deactivateApp(pAppId)
+  if not pAppId then pAppId = 1 end
+  common.getHMIConnection():SendNotification("BasicCommunication.OnAppDeactivated",
+    { appID = common.getHMIAppId(pAppId) })
+  common.getMobileSession(pAppId):ExpectNotification("OnHMIStatus")
+  :ValidIf(function(_, data)
+    if config["application"..pAppId].registerAppInterfaceParams.isMediaApplication == true then
+      return data.payload.audioStreamingState == "AUDIBLE" and
+        data.payload.videoStreamingState == "NOT_STREAMABLE" and
+        data.payload.hmiLevel == "LIMITED" and
+        data.payload.systemContext == "MAIN"
+    elseif config["application"..pAppId].registerAppInterfaceParams.appHMIType[1] == "NAVIGATION" then
+      return data.payload.audioStreamingState == "AUDIBLE" and
+        data.payload.videoStreamingState == "STREAMABLE" and
+        data.payload.hmiLevel == "LIMITED" and
+        data.payload.systemContext == "MAIN"
+    else
+      return data.payload.audioStreamingState == "NOT_AUDIBLE" and
+        data.payload.videoStreamingState == "NOT_STREAMABLE" and
+        data.payload.hmiLevel == "BACKGROUND" and
+        data.payload.systemContext == "MAIN"
+    end
+  end)
+end
+
+local function embeddedAudioActivated()
+  common.getHMIConnection():SendNotification("BasicCommunication.OnEventChanged",
+    { eventName = "AUDIO_SOURCE", isActive = true})
+  common.getMobileSession(1):ExpectNotification("OnHMIStatus",
+    { hmiLevel = "BACKGROUND", audioStreamingState = "NOT_AUDIBLE", videoStreamingState = "NOT_STREAMABLE" })
+  -- hmiLevel = "LIMITED", audioStreamingState = "AUDIBLE", videoStreamingState = "STREAMABLE"
```

Comment:
This is meant to be a comment indicating the state. Since it hasn't changed, no notification is sent. Perhaps I could expand the comment to be more clear on this matter?